.circleci/config.yml

```diff
@@ -126,6 +126,12 @@ jobs:
         TEST_VERBOSE: 1
     steps:
     - run: sudo apt update
+    - run: |
+        mkdir ~/localgo && cd ~/localgo
+        wget https://golang.org/dl/go1.16.6.linux-amd64.tar.gz
+        tar xfz go1.16.6.linux-amd64.tar.gz
+        echo "export PATH=$(pwd)/go/bin:$PATH" >> $BASH_ENV
+    - run: go version
```

Comment:
What's the reason why we're doing this/what happens when we don't? Is this because the machine executor doesn't have the same version of Go as the golang ones?

Reply:
Good question, this might be unnecessary code, it was carried over from https://github.com/ipfs/go-ipfs/pull/8138.  Let me dig deeper.

Reply:
Yeah this is required because the sharness test is using an Ubuntu image which is on Go 1.15.2, not the Go image.